pkg/kubectl/cmd/delete_test.go

```diff
@@ -307,10 +302,10 @@ func TestDeleteObjectGraceZero(t *testing.T) {
 	cmd := NewCmdDelete(fake, buf, errBuf)
 	cmd.Flags().Set("output", "name")
 	cmd.Flags().Set("grace-period", "0")
-	cmd.Run(cmd, []string{"pod/nginx"})
+	cmd.Run(cmd, []string{"pods/nginx"})
```

Comment:
Was this change required?

Reply:
> Was this change required?

fake mapper, fake typer, fake client.  One of them was unhappy.